tests/unit/utils.py

```diff
@@ -78,6 +77,9 @@ def project_from_dict(project, profile, packages=None, selectors=None, cli_vars=
 def config_from_parts_or_dicts(project, profile, packages=None, selectors=None, cli_vars='{}'):
     from dbt.config import Project, Profile, RuntimeConfig
     from copy import deepcopy
+    from dbt.config.utils import parse_cli_vars
+    if not isinstance(cli_vars, dict):
+        cli_vars = parse_cli_vars(cli_vars)
```

Comment:
This is now required because of [this change in feature/click-cli in the RuntimeConfig.from_parts](https://github.com/dbt-labs/dbt-core/pull/6396/files#diff-7685e44a07e8211f0e710116a07186168af0feb2e466fb46e40504d6b2282ec1R236) - RuntimeConfig now expects to be passed a parsed `vars` argument because this is done by click during parsing instead of on the fly!